### PR TITLE
feat(chore): implement yandex configuration in preset

### PR DESCRIPTION
**Pull request by artem-malko**
* Implement a basic configuration for Oauth2 with Yandex

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: examples/yandex.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const fastify = require('fastify')({ logger: { level: 'trace' } })
```
Comment (Eomm) [inline examples/yandex.js:1]: @fastify/plugins I think the examples/ folder is becoming too big to ship

I would switch to a single generic example and add a custom one only if necessary

Not necessary to do it in this PR tho 👍🏼

File: index.js [hunk 2]
```diff
@@ -322,6 +322,13 @@ fastifyOauth2.EPIC_GAMES_CONFIGURATION = {
   tokenPath: '/epic/oauth/v1/token'
 }
 
+fastifyOauth2.YANDEX_CONFIGURATION = {
+  authorizeHost: 'https://oauth.yandex.com',
+  authorizePath: '/authorize',
+  tokenHost: 'https://oauth.yandex.com',
+  tokenPath: '/token'
```
Comment (Eomm) [inline index.js:329]: Does it have a revoke url?

Comment (artem-malko) [reply index.js:329]: Yeap, it has.

POST request to `/revoke_token ` https://yandex.ru/dev/id/doc/en/tokens/token-invalidate#cleartext 

So I can add it right now.

Comment (artem-malko) [reply index.js]: And it is done)

File: examples/yandex.js (hunk 1 above)
Comment (Uzlopak) [inline examples/yandex.js:1]:
```suggestion
'use strict'

const fastify = require('fastify')({ logger: { level: 'trace' } })
```

File: index.js [hunk 3]
```diff
@@ -322,6 +322,17 @@ fastifyOauth2.EPIC_GAMES_CONFIGURATION = {
   tokenPath: '/epic/oauth/v1/token'
 }
 
+/**
+ * Yandex ID docs https://yandex.ru/dev/id/doc/en/
```
Comment (Uzlopak) [inline index.js:326]: Very good. I also thought that the lack of references is bad. Thank you for adding the reference.

**Review by Uzlopak — APPROVED** [review]
Over All LGTM

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**